packages/manager/apps/catalog/package.json

```diff
@@ -26,8 +26,6 @@
     "@ovh-ux/manager-core-api": "^0.2.0",
     "@ovh-ux/manager-react-core-application": "^0.2.2",
     "@ovh-ux/shell": "^3.0.5",
-    "@ovhcloud/msc-react-tile": "^1.1.2",
-    "@ovhcloud/msc-utils": "^0.2.1",
     "@ovhcloud/ods-common-theming": "16.0.1",
```

Comment:
No needed cause folder is in workspace and not builded

Reply:
It works because it's in a local environment
But we should declare all the package we import in the package.json though

Look around : all the @ovh-ux packages are in the workspace but we still declare them

Reply:
👍